resources/api/openapi.yml

```diff
@@ -0,0 +1,455 @@
+              type: boolean
+              example: false
+              description: If the user is a supporter of the angeltype.
+          required:
+            - confirmed
+            - supporter
+    Error:
+      type: object
+      properties:
+        message:
+          type: string
+    News:
+      type: object
+      properties:
+        id:
+          type: integer
+          example: 42
+        title:
+          type: string
+          example: First helper introduction
+        text:
+          type: string
+          example: |
+            The first introduction meeting takes place at the **Foo Hall** at 13:37.
+            Please bring your own seating as it might take some time.
+          description: |
+            The complete news text with markdown formatting.
+            Might include a `[more]` tag to be used as a separator which must be removed from the text when shown.
+        is_meeting:
+          type: boolean
+          example: true
+          description: Whether or not the news announces a meeting
+        is_pinned:
+          type: boolean
+          example: false
+          description: True if the news is pinned to the top
+        is_highlighted:
+          type: boolean
+          example: false
+          description: True if the news should be highlightet and shown on the dashboard
+        created_at:
+          type: string
+          format: date-time
+          description: DateTime in ISO-8601 format
+          example: 2023-05-13T13:37:42.000000Z
+        updated_at:
+          type: string
+          nullable: true
+          format: date-time
+          description: DateTime in ISO-8601 format
+          example: 2023-05-13T23:00:00.000000Z
+        url:
+          type: string
+          example: https://example.com/news/42
+          description: Direct link to the news page
+      required:
+        - id
+        - title
+        - text
+        - is_meeting
+        - is_pinned
+        - is_highlighted
+        - created_at
+        - updated_at
+        - url
+    Location:
+      type: object
+      properties:
+        id:
+          type: integer
+          example: 42
+        name:
+          type: string
+          example: Heaven
+        url:
+          type: string
+          example: https://example.com/location/42
+          description: Link of the location page
+      required:
+        - id
+        - name
+        - url
+    Shift:
+      type: object
+      properties:
+        id:
+          type: integer
+          example: 42
+        title:
+          type: string
+          example: Cleanup the venue
+        description:
+          type: string
+          example: You clean up the venue after the event, its fun, we promise!
+          description: |
+            Shift description, should be added to the shift type description as it might be empty.
+            Normally contains additional information for a specific shift / task.
+        start:
```

Comment:
I made good experience with using the `_at` suffix for date/time fields in general. It clarifies the expected type already
Hence, I recommend naming `start` -> `starts_at` and `end` - `ends_at`.

Reply:
Thats a good idea, i'll change that it